lib/matplotlib/_color_data.py

```diff
@@ -15,6 +15,24 @@
     'w': (1, 1, 1)}
 
 
+VEGA10_COLORS = {
+    'blue': '#1f77b4',
+    'orange': '#ff7f0e',
+    'green': '#2ca02c',
+    'red': '#d62728',
+    'purple': '#9467bd',
+    'brown': '#8c564b',
+    'pink': '#e377c2',
+    'gray': '#7f7f7f',
+    'olive': '#bcbd22',
+    'cyan': '#17becf'}
+
+
+# Normalize name to "vega:<name>" to avoid name collisions.
```

Comment:
I don't think there was a consensus on that (and I'm not sure it's necessary either.)